README.md

```diff
@@ -50,10 +50,7 @@ conda env create -f environment.yml
 This creates a new conda environment named `uafinfra` and installs
 _array_processing_ and all of its dependencies there.
 
-The final command above installs _array_processing_ in "editable" mode, which
-means that you can update it with a simple `git pull` in your local repository.
-We recommend you do this often, since this code is still under rapid
-development.
+The final command in the `environment.yml` file installs _array_processing_ in "editable" mode, which means that you can update it with a simple `git pull` in your local repository. We recommend you do this often, since this code is still under rapid development.
```

Comment:
Use "line" instead of "command" — also adjust line breaks for a smaller diff
```suggestion
The final line in the `environment.yml` file installs _array_processing_ in "editable" mode, which
means that you can update it with a simple `git pull` in your local repository.
We recommend you do this often, since this code is still under rapid
development.
```

Reply:
agreed